includes/iot-edge-verify-iot-edge-continuous-integration-continuous-deployment.md

```diff
@@ -30,4 +30,8 @@ To trigger a build job, you can either push a commit to source code repository o
 
 6. Click **dev** stage to see release logs.
 
-    ![Release logs](./media/iot-edge-verify-iot-edge-continuous-integration-continuous-deployment/release-logs.png)
\ No newline at end of file
+    ![Release logs](./media/iot-edge-verify-iot-edge-continuous-integration-continuous-deployment/release-logs.png)
+
+7. I need more help. My pipeline is failing.
+
+    Start by looking at the logs in your pipeline. You can view logs by navigating to the pipeline run summary and selecting the job and task. If a certain task is failing, check the logs for that task. For detailed instructions for configuring and using logs, see [Review logs to diagnose pipeline issues](https://docs.microsoft.com/en-us/azure/devops/pipelines/troubleshooting/review-logs?view=azure-devops).
```

Comment:
When you're referencing an article on docs that isn't in our repo, the easy way to reference them is to just remove "https://docs.microsoft.com" and the language parameter "en-us" from the URL. You also usually don't need to include the "?view=" parameter at the end, unless you're specifically referencing an older version of the product. But it looks like azure-devops is the default view. All that simplifies this link down to just:

`/azure/devops/pipelines/troubleshooting/review-logs`